src/features/people/PeopleListPage/index.js

```diff
@@ -64,7 +65,7 @@ const PeopleListPage = () => {
                   : `Popular people`
               }
             />
-            <Layout>
+            <PeopleContainer>
```

Comment:
can you remove Layout from imports then?

Reply:
Done! Thanks!